### PR TITLE
Release

### PR DESCRIPTION
**Pull request by tjholm**
Releases IfC for the python SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,42 +8,36 @@
...
 
-## Usage
+## Prerequisites
+
+- Python 3.8+
+
+## Getting Started
 
-### Nitric Functions (FaaS):
+### Using the [Nitric CLI](https://github.com/nitric-tech/cli)
```
Comment (asalkeld) [inline README.md:23]:
```suggestion
### Using the [Nitric CLI](https://github.com/nitrictech/cli)
```

File: nitric/resources/schedules.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        """
+        Register middleware to be run at the specified rate.
+
+        E.g. every("3 hours")
+        """
+        rate_description = rate_description.lower()
+
+        if not any([frequency in rate_description for frequency in Frequency.as_str_list()]):
+            # handle singular frequencies. e.g. every('day')
+            rate_description = f"1 {rate_description}s"  # 'day' becomes '1 days'
```
Comment (HomelessDinosaur) [inline nitric/resources/schedules.py:48]: This won't handle someone putting a frequency >1 with a singular descriptor e.g. `3 day`. But also might not be a problem.

File: nitric/resources/topics.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        return Events().topic(self.name)
+
+    def subscribe(self, func: EventMiddleware):
+        """Create and return a subscription decorator for this topic."""
+
+        self.server = FunctionServer(SubscriptionWorkerOptions(topic=self.name))
+        self.server.event(func)
+        Nitric._register_worker(self.server)
+
+        # return decorator
```
Comment (HomelessDinosaur) [inline nitric/resources/topics.py:108]: does this need to return the decorator?

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]

File: nitric/resources/schedules.py (hunk 2 above)
Comment (tjholm) [reply nitric/resources/schedules.py:48]: not handled in other SDKs at the moment, but can look at as a new issue.

File: nitric/resources/topics.py (hunk 3 above)
Comment (tjholm) [reply nitric/resources/topics.py]: fixed

**Review by asalkeld — APPROVED** [review]

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**